HWLesson2/Task5.js

```diff
@@ -0,0 +1,18 @@
+const checkForSpam = function(message) {
+  let check = false;
```

Comment:
Можна дещо спростити функцію :)
```suggestion
 if (message.toLowerCase().includes('spam')) {
    return true;
  }
  
  // dafault value
  return false;
```